src/js/views/eva-variant-view.js

```diff
@@ -214,14 +214,11 @@ EvaVariantView.prototype = {
 
     },
     _renderConsequenceTypeData: function (data) {
-
-        var annotation = data[0].annotation.consequenceTypes;
+        var _this = this;
+        var annotation = data[0].annotation.consequenceTypes.sort(_this._sortBy('ensemblGeneId', _this._sortBy('ensemblTranscriptId')));
```

Comment:
If using "this" will work well, the extra assignment doesn't add anything useful and could be removed.